### PR TITLE
Feature for popular menu

### PR DESCRIPTION
**Pull request by NancyRike**

### CODE REVIEW

File: src/pages/landingPage/LandingPage.js [hunk 1]
```diff
@@ -36,44 +43,44 @@ const LandingPage = () => {
...
+    //         description: 'Ora soup with fish and two beef and semo'
+    //     },
+    //     {
+    //         image: whiteRice,
+    //         name: 'White Rice',
+    //         price: '₦1,200',
+    //         description: 'White rice and stew with chicken'
+    //     },
+    //     {
+    //         image: okroSoup,
```
Comment (Dverybest) [inline src/pages/landingPage/LandingPage.js]: Remove unused code commented out

File: src/components/reusableMenuCard/Dish.js [hunk 2]
```diff
@@ -21,5 +23,10 @@ const Dish = ({ image, name, price, description }) => {
         </div>
     )
 }
-
+// const mapStateToProps = (state)=>{
```
Comment (Dverybest) [inline src/components/reusableMenuCard/Dish.js]: Remove unused commented out code

File: src/components/reusableMenuCard/Dish.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import './Dish.css'
 import {useHistory} from 'react-router-dom';
+// import { connect } from 'react-redux';
```
Comment (Dverybest) [inline src/components/reusableMenuCard/Dish.js]: Remove unused code commented out

File: src/pages/landingPage/LandingPage.js [hunk 4]
```diff
@@ -16,9 +16,16 @@ import okroSoup from '../../components/reusableMenuCard/dishImages/okroSoup.png'
 import Dish from '../../components/reusableMenuCard/Dish'
 import Card from '../../components/reusableCard/Card';
 import Footer from '../../components/footer/Footer';
+import { connect } from "react-redux";
 
-const LandingPage = () => {
+import {popularMenuAction} from "../../reduxSetup/actions/popularMenuAction";
+
+const LandingPage = (props) => {
+    console.log(props.popularMenuAction, "herehereHere");
```
Comment (Dverybest) [inline src/pages/landingPage/LandingPage.js]: Remove console.log

File: src/pages/landingPage/LandingPage.js [hunk 5]
```diff
@@ -163,4 +170,11 @@ const LandingPage = () => {
     )
 }
 
-export default LandingPage
+const mapStateToProps = (state)=>{
+    // console.log(popularMenu, 'looking for this');
```
Comment (Dverybest) [inline src/pages/landingPage/LandingPage.js]: Remove console.log

File: src/reduxSetup/actions/popularMenuAction.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import { requestProcessor } from "../../URLs/requestProcessor";
+import { POPULAR_MENU } from "../constants";
+// import { Dispatch } from "react";
+const popularMenuAction = (payload) => (dispatch) =>
+new Promise(async(resolve, reject) => {
+    try {
+        console.log("here?");
```
Comment (Dverybest) [inline src/reduxSetup/actions/popularMenuAction.js]: Remove console.log

File: src/reduxSetup/actions/popularMenuAction.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+            url: '/menu/getPopular',
+            dispatch,  
+        });
+        if (response.success) {
+            dispatch({
+                type: POPULAR_MENU,
+                payload: response.data
+            })
+            resolve(response);
+            console.log("you here?", response);
```
Comment (Dverybest) [inline src/reduxSetup/actions/popularMenuAction.js]: Remove console.log

**Review by Dverybest — CHANGES_REQUESTED** [review]
Looks good to me, just a little clean of console.logs and commented out codes required

File: src/pages/landingPage/LandingPage.js (hunk 4 above)
Comment (NancyRike) [reply src/pages/landingPage/LandingPage.js]: resolved

File: src/pages/landingPage/LandingPage.js (hunk 5 above)
Comment (NancyRike) [reply src/pages/landingPage/LandingPage.js]: fixed

File: src/reduxSetup/actions/popularMenuAction.js (hunk 6 above)
Comment (NancyRike) [reply src/reduxSetup/actions/popularMenuAction.js]: resolved

File: src/components/reusableMenuCard/Dish.js (hunk 3 above)
Comment (NancyRike) [reply src/components/reusableMenuCard/Dish.js]: this has been fixed

File: src/reduxSetup/actions/popularMenuAction.js (hunk 7 above)
Comment (NancyRike) [reply src/reduxSetup/actions/popularMenuAction.js]: this issue has been resolved

File: src/pages/landingPage/LandingPage.js (hunk 1 above)
Comment (NancyRike) [reply src/pages/landingPage/LandingPage.js]: resolved

**Review by Dverybest — APPROVED** [review]

**Pull request merged**